app/assignment/directives/instructorAssignmentTable.js

```diff
@@ -771,6 +796,10 @@ let instructorAssignmentTable = function ($rootScope, AssignmentActionCreators,
 			};
 
 			scope.renderTable();
+
+			if (scope.showStaffTable == true) {
```

Comment:
Update this too if you end up removing the `renderStaffTable()`. See comments above.